build.gradle

```diff
@@ -147,6 +148,13 @@ task wrapper(type: Wrapper) {
     gradleVersion = '2.3'
 }
 
+tasks.withType(FindBugs) {
```

Comment:
It is a single file `reports/findbugs/<env>.html`. I can be added as an individual tab in CI/CD servers such as Go.cd, Teamcity. I don't know if it's possible in Travis or Jenkins. Locally you can just point your browser to the file.